### PR TITLE
Checkout

### PR DESCRIPTION
**Pull request by OneExceptionn**
hey team, i made some updates so now the dropdown includes the 'all products' category, styling as been added to all products and single products view, checkout should be dynamic, but without the cart, it's difficult to test.

### CODE REVIEW

File: client/components/Navbar/index.js [hunk 1]
```diff
@@ -23,8 +23,7 @@ import {
   FormButton,
   NavLinkDown,
 } from "./NavbarElements";
-import CategoriesDropdown from "../Dropdown";
-import { render } from "react-dom";
+import Dropdown from "../Dropdown";
 
 class Navbar extends React.Component {
```
Comment (ysolovyova123) [inline client/components/Navbar/index.js:28]: this isn't the most recent version of Navbar/index.js that's in dev, pull from that one and resolve conflicts (most recent Navbar/index.js should have 2 different views, 1 for a user who is logged in vs. 1 for a user who is not.

File: client/store/categories.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ export const getCategories = () => {
                 uniqueCategories.push(category)
             }
         }))
+        console.log('thunk', uniqueCategories)
```
Comment (ysolovyova123) [inline client/store/categories.js]: remove the console.log

**Review by ysolovyova123 — CHANGES_REQUESTED** [review]
please see comments

File: client/store/categories.js (hunk 2 above)
Comment (OneExceptionn) [reply client/store/categories.js]: removed

**Review by ysolovyova123 — APPROVED** [review]

**Pull request merged**